packages/manager/modules/pci/src/projects/project/data-processing/notebooks/add-notebook/notebook-sizing/notebook-sizing.html

```diff
@@ -0,0 +1,102 @@
+<h4 data-translate="data_processing_add_notebook_notebook_title"></h4>
+<div class="row">
+    <!--Settings restricted-->
+    <oui-select-picker
+        data-ng-repeat="item in $ctrl.availableNotebookTemplate"
+        class="d-inline-block col-md-6 col-xl-3 my-3"
+        data-name="notebookSizing"
+        data-label="{{:: item.name}}"
+        data-model="$ctrl.selectedNotebookTemplate"
+        data-on-change="$ctrl.onChangeSelectedTemplate()"
+        data-values="[item.name]"
+    >
+        <oui-select-picker-section>
+            <p
+                class="mb-0"
+                data-translate="data_processing_add_notebook_template_notebook_cpu"
+                data-translate-values="{'nbCpu': item.CPU}"
+            ></p>
+            <p
+                data-translate="data_processing_add_notebook_template_notebook_ram"
+                data-translate-values="{'nbGio': item.RAM / 1024}"
+            ></p>
+        </oui-select-picker-section>
+        <oui-select-picker-section>
+            <ovh-manager-catalog-price
+                data-price="item.price"
+                data-tax="item.price"
+                data-user="$ctrl.user"
+                data-perform-rounding="false"
+                data-maximum-fraction-digits="5"
+            >
+            </ovh-manager-catalog-price>
+            /<span
+                data-translate="data_processing_add_notebook_price_unit"
+            ></span>
+        </oui-select-picker-section>
+    </oui-select-picker>
+</div>
+<h4 data-translate="data_processing_add_notebook_cluster_title"></h4>
+<p data-translate="data_processing_add_notebook_cluster_sizing_description"></p>
+<div class="row">
+    <!--Settings public-->
+    <oui-select-picker
+        data-ng-repeat="item in $ctrl.availableClusterTemplate track by item.id"
+        class="d-inline-block col-md-6 col-xl-3 my-3"
+        data-name="clusterSizing"
+        data-label="{{:: item.name}}"
+        data-model="$ctrl.selectedClusterTemplate"
+        data-on-change="$ctrl.onChangeSelectedTemplate()"
+        data-values="[item.name]"
+    >
+        <oui-select-picker-section>
+            <p
+                class="mb-0"
+                data-translate="data_processing_add_notebook_template_driver_cpu"
+                data-translate-values="{'nbCpu': item.driverCores}"
+            ></p>
+            <p
+                class="mb-0"
+                data-translate="data_processing_add_notebook_template_driver_ram"
+                data-translate-values="{'nbGio': item.driverMemory / 1024}"
```

Comment:
i guess it better to create a function to make this compute.
```suggestion
                data-translate-values="{'nbGio': $ctrl.convertToGio(item.driverMemory)}"
```

Reply:
Fixed :)